tagstudio/src/qt/ts_qt.py

```diff
@@ -525,11 +542,37 @@ def start(self):
                 QColor("#9782ff"),
             )
             self.open_library(lib)
-
         app.exec_()
 
         self.shutdown()
 
+    def openSettings(self):
```

Comment:
I know it's difficult to pick the style since PySide is using camelCase, but please try to follow PEP-8 wherever possible as mentioned in README.

```suggestion
    def open_settings(self):
```